README.md

```diff
@@ -1,3 +1,59 @@
-## My test task
+# My test task
+
+---
 
 Here you can find my implementation of OrderBook.
+
+---
```

Comment:
Add the `Background` section that will explain the real world problem.
And show, how you solution fix it.